### DISCUSSION TITLE
tutorials/editor/default_key_mapping

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#289] [User-contributed notes]
# tutorials/editor/default_key_mapping

Many Godot editor functions can be executed with keyboard shortcuts. This page lists functions which have associated shortcuts by default, but many others are available for customization in editor ...

https://docs.godotengine.org/en/4.3/tutorials/editor/default_key_mapping.html

**Comment by MartinDelille** [comment]
The Op+1/2/3 shortcut doesn't switch between 2D/3D/Script editor on my Macbook (without numpad).

**Reply by Calinou** [reply]
Which keyboard layout are you using?

**Reply by MartinDelille** [reply]
English

**Reply by MartinDelille** [reply]
I mean US keyboard.

**Reply by MartinDelille** [reply]
From what I read here the documentation is incorrect: https://github.com/godotengine/godot/blob/4.3/editor/editor_node.cpp#L7831-L7834

**Reply by MartinDelille** [reply]
I opened a PR here: https://github.com/godotengine/godot-docs/pull/10575

**Comment by AnidemDex** [comment]
Just if anyone wondering, there's no way (yet) to get any of these shortcuts by code
